library/alloc/src/string.rs

```diff
@@ -1612,6 +1612,81 @@ impl String {
         &mut self.vec
     }
 
+    /// Returns a raw pointer to the string's buffer.
+    ///
+    /// The caller must ensure that the string outlives the pointer this
+    /// function returns, or else it will end up pointing to garbage.
+    /// Modifying the string may cause its buffer to be reallocated,
+    /// which would also make any pointers to it invalid.
+    ///
+    /// The caller must also ensure that the memory the pointer (non-transitively) points to
+    /// is never written to (except inside an `UnsafeCell`) using this pointer or any pointer
```

Comment:
```suggestion
    /// is never written to using this pointer or any pointer
```
There is no `UnsafeCell` here.